pkg/client/clientcmd/loader.go

```diff
@@ -63,9 +64,16 @@ func (rules *ClientConfigLoadingRules) Load() (*clientcmdapi.Config, error) {
 	config := clientcmdapi.NewConfig()
```

Comment:
Add a note to the function doc mentioning that relative paths in files are resolved relative to the referencing file